account/manager.go

```diff
@@ -107,6 +108,70 @@ func (wt witnessType) witnessSize() (int, error) {
 	}
 }
 
+// IsExpirySpend returns true if the witness is taking an expiration path.
+func (wt witnessType) IsExpirySpend() bool {
```

Comment:
This should be moved to where the `witnessType` is declared.

Reply:
`witnessType` is declared within this file here: https://github.com/lightninglabs/pool/pull/396/files/d76f0f11114ba07e0d54a63ca9d233f37a8f2286#diff-94465ef38c605b8bddd6e99ee32de13f57b8d6135d6be738f750498e82f19c8eR64

Above the `IsExpirySpend()` declaration are other `witnessType` methods:
```go
func (wt witnessType) scriptVersion() poolscript.Version {...}
...
func (wt witnessType) witnessSize() (int, error) {...}
```

Perhaps constants defined in between lines 71 and 81 should be moved elsewhere, is that what you mean? Let me know if there's a better place for `IsExpirySpend()` or if I've missed something.

Reply:
Oh, I'm sorry, that's my fault... I thought the `witnessType` was declared in a different file (but the same package). Should've clicked the arrow on the diff to show more of the code before L111. Never mind :see_no_evil:

Reply:
No problem! Review tool has room for improvement to be honest. Not sure if I know of better.